ui/apps/platform/cypress/integration/policies/policiesTable.test.js

```diff
@@ -183,7 +208,8 @@ describe('Policies table', () => {
                     cy.wait('@getPolicies');
 
                     // Policy table
-                    cy.get(`h1:contains("Policies")`);
+                    cy.get(`.pf-c-title:contains('Policy Management')`);
+                    cy.get(`.pf-c-nav__link.pf-m-current:contains("Policies")`);
```

Comment:
@pedrottimark I seem to remember that you created helper functions for "top matter" stuff like this? If that's so, please share with Linda, so we can start using in the next PR.

Reply:
* Aha, there is a helper function to **visit and assert**.
* Here is assert only after click **Cancel** so seems okay inline for one occurrence. Can factor out to helpers, if and when multiple tests need similar assertion.

Reply:
Oh, there is another occurrence below, so an `assertWhatever` helper would be a nice follow up.